report a bug!!!

pkg/naming/discovery/discovery.go:282 +0x152

		var appIns *naming.InstancesInfo
		appIns, ok = app.zoneIns.Load().(*naming.InstancesInfo)
		ins = new(naming.InstancesInfo)
		ins.LastTs = appIns.LastTs
		ins.Scheduler = appIns.Scheduler

ins can be nil
panic: runtime error: invalid memory address or nil pointer dereference

--- a/pkg/naming/discovery/discovery.go
+++ b/pkg/naming/discovery/discovery.go
@@ -278,6 +278,9 @@ func (r *Resolve) Fetch(ctx context.Context) (ins *naming.InstancesInfo, ok bool
 	if ok {
 		var appIns *naming.InstancesInfo
 		appIns, ok = app.zoneIns.Load().(*naming.InstancesInfo)
+		if !ok {
+			return
+		}
 		ins = new(naming.InstancesInfo)
 		ins.LastTs = appIns.LastTs
 		ins.Scheduler = appIns.Scheduler

